resources/web/README

```diff
@@ -0,0 +1 @@
+template.html is fetched by fetch_template.sh 
```

Comment:
If this is a manual step, I'd say "_can be fetched"_.

"_is fetched_" implies that it happens automagically.

Reply:
++

Reply:
Done!